examples/getitem/pyproject.toml

```diff
@@ -0,0 +1,16 @@
+[build-system]
+requires = ["maturin>=0.13,<0.14"]
```

Comment:
I think Maturin just reached 1.0 and we should use that current version here.

Reply:
Updated to 1